src/styles/transitiongroup.scss

```diff
@@ -0,0 +1,46 @@
+.item-enter {
```

Comment:
Seems like in `src/styles/todo.scss` you have the same styles but with other [nesting](https://mate.academy/learn/css-advanced/sass#/video/925)

`-` in `item-enter-active` just a separator for words it doesn't mean modifier of element. Rewatch [naming contention in BEM](https://mate.academy/learn/css-advanced/bem#/video/920)